dev/docker-compose.dev.yaml

```diff
@@ -30,6 +30,18 @@ services:
         networks:
             - nango
 
+    temporal-ui:
+        image: temporalio/ui:2.26.0
+        depends_on:
+            - temporal
+        environment:
+            - TEMPORAL_ADDRESS=temporal:7233
+            - TEMPORAL_CORS_ORIGINS=http://localhost:3000
+        ports:
+            - 8081:8080
```

Comment:
why not the same port?

Reply:
8080 is very often used locally, I just wanted to avoid conflict